test/variants.js

```diff
@@ -30,6 +30,8 @@ describe('variants', () => {
       'group/llama',
       'group-hover/llama:m-32',
       'part-[part-name]:s-bg-primary',
+      'data-[is-active]:s-bg-primary',
+      'aria-[invalid=spelling]:s-bg-primary',
```

Comment:
this test doesn't produce any valid css. Suggest to add 2 separate tests like:

 ```
 test('data variant', async ({ uno }) => {
    const classes = ['data-[is-active]:s-bg-primary'];
    const { css } = await uno.generate(classes);
    expect(css).toMatchSnapshot();
  });

  test('aria variant', async ({ uno }) => {
    const classes = ['aria-[invalid=spelling]:s-bg-primary'];
    const { css } = await uno.generate(classes);
    expect(css).toMatchSnapshot();
  });
```

Reply:
👍 Done.